x-pack/plugins/ml/server/routes/inference.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RouteInitialization } from '../types';
+import { wrapError } from '../client/error_wrapper';
+import { modelIdSchema, optionalModelIdSchema } from './schemas/inference_schema';
+
+export function inferenceRoutes({ router, mlLicense }: RouteInitialization) {
+  /**
+   * @apiGroup Inference
+   *
+   * @api {get} /api/ml/inference/:modelId Get info of a trained inference model
+   * @apiName GetInferenceModel
+   * @apiDescription Retrieves configuration information for a trained inference model.
+   */
+  router.get(
+    {
+      path: '/api/ml/inference/{modelId?}',
+      validate: {
+        params: optionalModelIdSchema,
+      },
+      options: {
+        tags: ['access:ml:canAccessML'],
```

Comment:
using `canAccessML` places these endpoints as basic license features. Is that correct?
should they be behind a DFA capability?

Reply:
As discussed changed the privileges check to `access:ml:canGetDataFrameAnalytics` in https://github.com/elastic/kibana/pull/74978/commits/5c4dcc4480f9c6793bbf70f6e7e170f9f91a7273